### ISSUE TITLE
Having issues with compressed files 

### ISSUE THREAD

**Issue by TheJoe49** [happypandax/happypandax#156]
**Describe the bug**
When I go to scan for new galleries, having "Scan only for new galleries" and "Move Galleries" selected, I get duplicates of the ZIP galleries and it is also unable to move them. HappypandaX can move and skip the regular galleries in a folder, and I can view the zip galleries just fine. Its just when it comes to not adding a duplicates and moving the ZIP file to a new folder after the scan, it fails. 

I have tried a new install. 
The server and the files are on the same drive, just not the main drive.

**Expected behavior**
For happypandax to move compressed files and to not add duplicates of them in the browser. 

**Logs**
[error.log](https://github.com/happypandax/happypandax/files/3593223/error.log)
[plugin.log](https://github.com/happypandax/happypandax/files/3593210/plugin.log)
[activity.log](https://github.com/happypandax/happypandax/files/3593228/activity.log)

The Debug log was to big, ~~here is the paste bin~~ ,too big for paste bin here is a drive link :  https://drive.google.com/open?id=1dswQx7Sr6XGfoMY1yRm-TCgalLlFFB-Z

**Environment:**
 - OS: Windows 10
 - Browser and browser version : Chrome
 - HPX Version : 11.2

**Comment by Stealthbird97** [comment]
I seem to have very similar problems. I scan a directory using "Scan only for new galleries", it finds galleries, and I submit them to the inbox.
Scan the same directory for a second time with the same settings, finds the same galleries as before, and will duplicate when submitted to the inbox.

[error.log](https://github.com/happypandax/happypandax/files/3602718/error.log)

I've also noticed for some reason the log entries are showing Sept-09, rather than Sept-11, on errors occurring now...

**Environment:**
OS: Windows 10 Enterprise
HPX Server Version : 0.11.2
HPX Webclient Version : 0.11.1
HPX db Version : 0.7.0

**Comment by twiddli** [comment]
Positive I fixed this in `v0.12.0`, if not, please re-open.

**Comment by TheJoe49** [comment]
First, I'm so sorry to bring this back when you thought you were done.

I am having the same issue with adding duplicate galleries and a new one when trying move them on import. In order to help find out whats happening I recorded from installation to the issue and I think I found an issue with adding zip files.

In my video, I try adding two zip files and a regular folder as a gallery and when I try scanning the galleries again it only duplicates one of the two zip files. The only difference that I found was that one zip had a folder within it. So my guess is that the issue is when importing Zip files with a folder inside that then has the image files.

I also tried this with the PostgreSQL database and same thing happened.

Recording: [VIDEO](https://youtu.be/C7l7-O2P4Jw)
Logs Folder: [Drive Link](https://drive.google.com/drive/folders/1uS_3r0yjvMn6uAOfpqSdZ9pZVPTHSDS5?usp=sharing)
